plugins/main/common/constants.ts

```diff
@@ -55,6 +55,10 @@ export const VULNERABILITY_IMPLICIT_CLUSTER_MODE_FILTER = 'wazuh.cluster.name';
 export const WAZUH_FIM_PATTERN = 'wazuh-states-fim-*';
 export const WAZUH_FIM_FILES_PATTERN = 'wazuh-states-fim-files-*';
 export const WAZUH_FIM_REGISTRIES_PATTERN = 'wazuh-states-fim-registries-*';
```

Comment:
I understand that this could be removed, right?

Reply:
Solved here: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7604/commits/1776bca244db90bfd226f6d13a2d85a550b79a0e